### PR TITLE
mention lisp-repl-core-dumper for instant SLY startup.

### PR DESCRIPTION
**Pull request by Ambrevar**
* README.md (Instant REPL startup): New section.

* doc/sly.texi (Loading Slynk faster): Mention lisp-repl-code-dumper.

### CODE REVIEW

**Comment by Ambrevar** [comment]
Supersedes #366.

**Comment by joaotavora** [comment]
Pierre, I don't mean to belittle your good work, but I think this is still too wordy and distracting.  I want a focused README and manual that explains what SLY can do, with occasional tangents for in-depth exploration elsewhere. There no need to justify the "why" of your project in SLY's README, a "to know more" link suffices.  Same things with contribs, they are merely mentioned by link.  The link name enough is enough of an ad.  So here's my proposal:

```diff
diff --git a/README.md b/README.md
index 5a3ba40c..13337461 100644
--- a/README.md
+++ b/README.md
@@ -100,6 +100,12 @@ $ sbcl
 Now in Emacs you can do `sly-connect` and give it the host and the 4008 port as
 a destination.
 
+Faster startup
+--------------
+
+If the Lisp program doesn't start fast enough for you, look in [the
+manual][instasly], for ways to make it faster.
+
 Additional Contribs
 -------------------
 
@@ -174,6 +180,7 @@ first part [CONTRIBUTING.md][5] file for instructions on how to contribute.
 [14]: https://github.com/company-mode/company-mode
 [15]: https://github.com/emacs-helm/helm-sly
 [documentation]: http://joaotavora.github.io/sly
+[instasly]: http://joaotavora.github.io/sly/#Loading-Slynk-faster
 [tutorial]: http://joaotavora.github.io/sly/#A-SLY-tour-for-SLIME-users
 
 <!-- Local Variables: -->
diff --git a/doc/sly.texi b/doc/sly.texi
index e1c90c00..0e513b50 100644
--- a/doc/sly.texi
+++ b/doc/sly.texi
@@ -2899,6 +2899,10 @@ In this section, a technique to load Slynk faster on South Bank Common
 Lisp (SBCL) is presented.  Similar setups should also work for other
 Lisp implementations.
 
+A pre-canned solution that automates this technique was developed by
+@uref{https://gitlab.com/ambrevar/lisp-repl-core-dumper,Pierre
+Neidhardt}.
+
 For SBCL, we recommend that you create a custom core file with socket
 support and @acronym{POSIX} bindings included because those modules
 take the most time to load.  To create such a core, execute the
```

**Comment by Ambrevar** [comment]
No need to apologize, I perfectly understand!  Your patch looks good to
me, feel free to merge!

**Comment by Ambrevar** [comment]
Thanks!
